doc/architecture/performance/submitter.rst

```diff
@@ -0,0 +1,57 @@
+Submitter
+=========
+
+Overview
+--------
+
+The Submitter component is written in C++ and submits multiple requests with a very high speed to
+stress test a system. In order to run the submitter, the required libraries should 
+be installed. The following command in the ``CCF/getting-started/setup_vm`` directory installs them:
+
+.. code-block:: bash
+
+    $ ./run.sh ccf-dev.yml

```

Comment:
Maybe linking to the existing docs page on setting up a vm for CCF would be better. Less duplications makes it easier to maintain the docs going forward